digitalaidseattle/games-for-love
# Sushmita/hospital landing page for past hospitals

## Description

Basic Layout For Hospital Landing Page

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

- Related Issue # https://projects.digitalaidseattle.org/project-tasks-details?recordId=rec8b0rj6ZXujxE5j
- Closes # https://projects.digitalaidseattle.org/project-tasks-details?recordId=rec8b0rj6ZXujxE5j

## QA Instructions, Screenshots, Recordings
This hospital page will be available on click on "Learn More" on any hospital. The past & the active hospital have only the "Donation" section as different will be handled with hospital context status in future.

Few Notes : This is still a WIP. I am still mapping the hospitalContext for many sections. For now this is static layout with texts. Few sections are dynamic for now. I am mapping for most of the other sections.

What to Test: 
I wanted to get my component structure reviewed. Also any feedback on how to make the sections more reusable will be great!

Figma Expected:
![image](https://github.com/user-attachments/assets/cfc57496-5c70-414f-b97d-e89eae9693d7)

How it looks for now:
![image](https://github.com/user-attachments/assets/e5c37874-e4bb-41d1-9464-dea8ca8571f5)
